librosa/sequence.py

```diff
@@ -367,7 +383,7 @@ def __dtw_calc_accu_cost(C, D, D_steps, step_sizes_sigma,
 
 
 @jit(nopython=True, cache=True)
-def __dtw_backtracking(D_steps, step_sizes_sigma, subseq):  # pragma: no cover
+def __dtw_backtracking(D_steps, step_sizes_sigma, subseq, start_m=None):  # pragma: no cover
```

Comment:
why `start_m` and not just `start`?

Reply:
My idea was that this is the start index for the column (instead of `M-1`). The start index for the row is still `N-1`. But I agree that this idea is not obvious and it may be better to just call it `start`.